src/views/blocks/tickets.php

```diff
@@ -20,10 +20,30 @@
 $cart_url     = $this->get( 'cart_url' );
 $cart_classes = array( 'tribe-block', 'tribe-block__tickets' );
 
-// We don't display anything if there is not provider
-if ( ! $provider ) {
+// We don't display anything if there is no provider or tickets
+if ( ! $provider || empty( count( $tickets ) ) ) {
```

Comment:
Using empty() like this is PHP 5.5+ only (empty only supports variables prior to 5.5)